plugin/parser/differ/mysql/differ.go

```diff
@@ -105,5 +106,35 @@ func buildTableMap(nodes []ast.StmtNode) map[string]*ast.CreateTableStmt {
 
 // isTwoColumnsSame returns true if the two columns are the same.
 func isTwoColumnsSame(old, new *ast.ColumnDef) bool {
```

Comment:
This function name bothers me. Generally, `isTwoColumnsSame` also needs to check the other column definition, such as the column names.
If we only want to check the column options here, I think `sameColumnOptions` is better.

Reply:
We had compared the column type in this function. But extracting a function to compare the column options can make this function more clarify.

Reply:
How about also checking the column name in this function?